angular.json

```diff
@@ -60,13 +60,15 @@
             ],
             "scripts": [
               "node_modules/prismjs/prism.js",
+              "node_modules/prismjs/components/prism-sql.min.js",
               "node_modules/prismjs/components/prism-csharp.min.js",
               "node_modules/prismjs/components/prism-css.min.js",
```

Comment:
Now the question is do we need CSharp? :-)

Reply:
Removed, no, it's not necessary